crates/rover-client/src/query/schema/get.rs

```diff
@@ -0,0 +1,108 @@
+use crate::blocking::Client;
```

Comment:
This file is also updated in this PR: https://github.com/apollographql/rover/pull/50/files#diff-a3d96ce2322c31552b8ebaf2f6ace8b71024b2070d5eca6a17c99c01904128d9